src/index.js

```diff
@@ -181,7 +201,11 @@ export default class Camera extends Component {
     const style = [styles.base, this.props.style];
     const nativeProps = convertNativeProps(this.props);
 
-    return <RCTCamera ref={CAMERA_REF} {...nativeProps} />;
+    if(this.state.isAuthorized) {
```

Comment:
can we have something like this one https://github.com/moaazsidat/react-native-qrcode-scanner/blob/master/index.js#L203

`pendingAuthorizationView` and
`notAuthorizedView`